docs/customization.rst

```diff
@@ -118,6 +118,24 @@ Unlike, newer versions of mkdocs-material theme, this theme also supports the us
 
         :hero: Configuration options to personalize your site.
 
+.. confval:: hide
+
+    Unlike the mkdocs-material theme, using the ``:hide:`` metadata can only be used to hide the
+    page's navigation menu (on large viewports only). This is done like so:
+
+    .. code-block:: rst
+
+        :hide: navigation
+
+    .. note::
+
+        Instead of using ``:hide: toc``, this theme uses the ``:tocdepth:`` metadata to hide the
+        page's Table of Contents.
+
+        .. code-block:: rst
+
+            :tocdepth: 0
```

Comment:
I also noticed that tocdepth is not actually respected by nav_adapt.py, other than handling the special case of `tocdepth == 0`.

Possibly we could support `:hide: toc` as well for consistency with mkdocs and since it is clearer than having to specify `tocdepth`.

Reply:
I guess one challenge with `:hide:` as a field element is that it seems that sphinx creates a dictionary --- is there a way to support multiple `:hide:` metadata values?

Reply:
>is there a way to support multiple :hide: metadata values?

This is why I chose to keep the implemented tocdepth metadata. I suppose we could do
```py
hide_list = meta.get("hide").split(",")
if "navigation" in hide_list:
    # ...
if "toc" in hide_list:
    # ...
```

Reply:
Actually I think we should go with something like `:hide-navigation:` rather than `:hide: navigation` --- that way we are more consistent with how these document metadata fields are normally treated (as key-value assignments).

Reply:
Would you like to preserve the `tocdepth` option for backward compatibility? Or do you consider it broken enough to get rid of it?

Reply:
> metadata fields are normally treated (as key-value assignments).

So what value do you propose? A bool? Or does sphinx use a default value in case none is specified?

Reply:
I think they are just parsed as strings, but for bool flags the convention seems to be to just leave the value blank:

https://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html#special-metadata-fields

So just:

```rst
:hide-navigation:
```

As far as tocdepth --- thinking about it more, I think we should just keep using tocdepth for this purpose, rather than introducing a new `:hide-toc:` field.  At some point we should actually make non-zero values of `tocdepth` have their proper effect, but for now it seems reasonable to leave it as is.

I think currently, on the master document only the toc depth is limited to 2, and on all other pages the toc depth is unlimited.  This difference is due to the fact that nav_adapt.py uses different logic in those two cases for obtaining the local toc.

Reply:
I can implement both `hide-toc` while preserving `tocdepth`. In testing the new metadata idea, the users would have to specify 
```rst
:hide-naviagation: anyValue
```
otherwise, no value has no effect.
```rst
.. this has no effect
:hide-naviagation:
```

Reply:
NM, I got it to work if the metadata is specified at all. If it isn't there, then the feature is not enabled for that page.

Reply:
I'm going forward with the `hide-toc` idea because it better matches mkdocs-material convention (& its more descriptive). People migrating from mkdocs to sphinx may get confused about the toctree directive's option `maxdepth`. To be clear, `:tocdepth: 0` will still work (just like `hide-toc`).